.editorconfig

```diff
@@ -15,6 +15,7 @@ max_line_length = 88
 ensure_newline_before_comments = true
 include_trailing_comma = true
 use_parentheses = true
+trim_trailing_whitespace = false
```

Comment:
I object to this change. We can put a subsidiary `.editorconfig` in `test/Interactive` to solve the specific problem. The whitespace stripping in general on Python files is valuable.